src/js/VideoStorage.js

```diff
@@ -0,0 +1,24 @@
+import { ERROR_MESSAGE, STORAGE_MAX_COUNT } from "./constants/constants";
+
+export default class VideoStorage {
+  constructor() {
+    this.storage = JSON.parse(localStorage.getItem("videos")) || [];
```

Comment:
```suggestion
    this.videos = JSON.parse(localStorage.getItem("videos")) || [];
```
videos가 더 어울린다고 생각되요 👀